src/main/java/at/hannibal2/skyhanni/features/misc/limbo/LimboPlaytime.kt

```diff
@@ -0,0 +1,121 @@
+package at.hannibal2.skyhanni.features.misc.limbo
+
+import at.hannibal2.skyhanni.SkyHanniMod
+import at.hannibal2.skyhanni.events.InventoryOpenEvent
+import at.hannibal2.skyhanni.events.LorenzToolTipEvent
+import at.hannibal2.skyhanni.utils.LorenzUtils
+import at.hannibal2.skyhanni.utils.LorenzUtils.round
+import at.hannibal2.skyhanni.utils.NEUItems
+import at.hannibal2.skyhanni.utils.NumberUtil.addSeparators
+import at.hannibal2.skyhanni.utils.StringUtils.matches
+import at.hannibal2.skyhanni.utils.repopatterns.RepoPattern
+import io.github.moulberry.notenoughupdates.events.ReplaceItemEvent
+import io.github.moulberry.notenoughupdates.util.Utils
+import net.minecraft.client.player.inventory.ContainerLocalMenu
+import net.minecraftforge.fml.common.eventhandler.SubscribeEvent
+import kotlin.time.Duration.Companion.seconds
+
+class LimboPlaytime {
+    private lateinit var modifiedList: MutableList<String>
+    private var setMinutes = false
+    private val patternGroup = RepoPattern.group("misc.limbo.tooltip")
+    private val minutesPattern by patternGroup.pattern(
+        "minutes",
+        "§5§o§a([\\d.,]+) minutes.+\$"
+    )
+    private val hoursPattern by patternGroup.pattern(
+        "hours",
+        "§5§o§b([\\d.,]+) hours.+\$"
+    )
+
+    private var wholeMinutes: Long = 0
+    private var hoursString: String = ""
+
+    private val config get() = SkyHanniMod.feature.misc
+
+    @SubscribeEvent
+    fun replaceItem(event: ReplaceItemEvent) {
+        if (event.inventory !is ContainerLocalMenu) return
+        if (event.inventory.displayName.unformattedText != "Detailed /playtime") return
+        if (event.slotNumber != 43) return
+        if (config.limboPlaytime == 0) return
+
+        val limboItem by lazy {
```

Comment:
using by lazy here does not help.
Please cache the item (not the lore) with a member variable and "by lazy" instead of creating it on every replace item event call.

Reply:
Then Build the final item stack in this function, but also cache it every second or so